### PR TITLE
Return soft alignment in Nematus style format

### PR DESCRIPTION
**Pull request by M4t1ss**
Use parameter `--return-nematus-alignment` (together with `--no-debpe`) to return soft alignments just like in Nematus with `--output_alignment` or `-a`.
For each input sentence write 

> \<sentence id> ||| \<target words> ||| \<score> ||| \<source words> ||| \<number of source words> ||| \<number of target words>
> soft alignment matrix
> newline

### CODE REVIEW

**Comment by hieuhoang** [comment]
thanks. Regression tests passed. I'll pull it unless anyone has objections

File: src/amun/common/translation_task.cpp [hunk 1]
```diff
@@ -21,7 +21,7 @@ void TranslationTaskAndOutput(const God &god, std::shared_ptr<Sentences> sentenc
     size_t lineNum = history.GetLineNum();
 
     std::stringstream strm;
-    Printer(god, history, strm);
+    Printer(god, history, strm, *sentences);
```
[the remark this answers is not in the document]
Comment (M4t1ss) [reply src/amun/common/translation_task.cpp]: Would it be safer to get the specific `sentence` and pass that just like it is done with `history`?
```C++
    const History &history = *histories->at(i);
    size_t lineNum = history.GetLineNum();
    const Sentence &sentence = *sentences->at(0);

    std::stringstream strm;
    Printer(god, history, strm, sentence);
```

[the remark this answers is not in the document]
Comment (tomekd) [reply src/amun/common/translation_task.cpp]: Yeah, it's better.

[the remark this answers is not in the document]
Comment (M4t1ss) [reply src/amun/common/translation_task.cpp]: OK, updated ⚙️

**Pull request merged**